### DISCUSSION TITLE
Updated UI

### DISCUSSION THREAD

**Discussion by spiraloid** [carson-katri/dream-textures#449] [General]
Heads up, the concept artists and their fans are aware you put their names in the ui. Lots of pitch fork sharpening.  you might want to amend the enum or make a preference. 

![image](https://user-images.githubusercontent.com/36362743/208317607-7166dfe5-f6f3-4277-af45-9f2a15a9e80e.jpeg)

**Comment by NullSenseStudio** [comment]
Concept artist names were already removed over two months ago in version 0.0.6

71882df47b9da4b73dc3c365d65ee10d27c87223

**Reply by spiraloid** [reply]
Excellent
